### PR TITLE
Adding Common Lab Module

### PR DESCRIPTION
**Pull request by seekme94**
I have followed the [module release guide](https://wiki.openmrs.org/display/docs/Module+Release) on Wiki. I can see a published version on Bintray.
At present, this module is parked in my Bintray account, but in the future will be managed under OpenMRS.

### CODE REVIEW

**Comment by dkayiwa** [comment]
@reubenvarghese1 what do you think of this?

File: src/main/resources/add-ons-to-index.json [hunk 1]
```diff
@@ -783,6 +783,41 @@
...
+        }
+      ],
+      "links": [
+        {
+          "rel": "documentation",
+          "href": "https://wiki.openmrs.org/display/docs/Common+Lab+Module",
+          "title": "Wiki Page"
+        },
+        {
+          "rel": "documentation",
```
Comment (reubenvarghese1) [inline src/main/resources/add-ons-to-index.json]: I am not sure if we can have multiple documentation links. So, to be on the safer side, I would request you to remove either of the links. I would have probably removed the link to the API and added that link within the documentation of the module.

File: src/main/resources/add-ons-to-index.json [hunk 2]
```diff
@@ -783,6 +783,41 @@
...
+          "title": "REST API"
+        },
+        {
+          "rel": "source",
+          "href": "https://github.com/openmrs/openmrs-module-commonlabtest"
+        }
+      ],
+      "backend": "org.openmrs.addonindex.backend.Bintray",
+      "bintrayPackageDetails": {
+        "owner": "seekme94",
```
Comment (reubenvarghese1) [inline src/main/resources/add-ons-to-index.json]: TLDR: Kindly change the owner to openmrs (I am aware that your module is not in the openmrs account)

We only have a certain quota for using the bintray api to fetch module details from external accounts(non-openmrs bintray accounts). Hence, to not exhaust the quota, I will be linking to your module from the OpenMRS Bintray account. Kindly check [this](https://talk.openmrs.org/t/clarification-needed-on-adding-modules-to-bintray/18613/3) discussion for further context

**Comment by seekme94** [comment]
Hi @reubenvarghese1, I've made the suggested changes. The link to REST API has been removed and the owner is changed to "openmrs".

**Comment by reubenvarghese1** [comment]
@seekme94 , your module doesn't seem to have an omod uploaded. It's under the maven category. Addons indexes the module and requires a download link to the owa and omod. 

@dkayiwa can you please verify whether I'm right?  Here's the link to the module https://bintray.com/seekme94/maven/commonlabtest/1.4.0

**Comment by seekme94** [comment]
> @seekme94 , your module doesn't seem to have an omod uploaded. It's under the maven category. Addons indexes the module and indexrs a download link to the owa and omod. Infact, even in the openmrs bintray account, all the modules are either an omod or an owa. There's only one package under the maven category i.e. Openmrs SDK package ( which is not a module and isn't indexed by addons )
> 

@reubenvarghese1 you'll find the omod file in Files. Unless, of course I missed something out.

**Comment by seekme94** [comment]
> @seekme94 , your module doesn't seem to have an omod uploaded. It's under the maven category. Addons indexes the module and indexrs a download link to the owa and omod. Infact, even in the openmrs bintray account, all the modules are either an omod or an owa. There's only one package under the maven category i.e. Openmrs SDK package ( which is not a module and isn't indexed by addons )
> 
> @dkayiwa can you please verify whether I'm right? Here's the link to the module https://bintray.com/seekme94/maven/commonlabtest/1.4.0
> 
> The pr looks good to me so thanks for that! :)

Got it. Gonna fix this, I also think there's some tuning required on Wiki guide. Will see if I can update that as well.

**Comment by seekme94** [comment]
> @seekme94 , your module doesn't seem to have an omod uploaded. It's under the maven category. Addons indexes the module and indexrs a download link to the owa and omod. Infact, even in the openmrs bintray account, all the modules are either an omod or an owa. There's only one package under the maven category i.e. Openmrs SDK package ( which is not a module and isn't indexed by addons )
> 
> @dkayiwa can you please verify whether I'm right? Here's the link to the module https://bintray.com/seekme94/maven/commonlabtest/1.4.0
> 
> The pr looks good to me so thanks for that! :)

Fixed it. The Repo is now Generic omod like other modules. Here:
https://bintray.com/seekme94/omod/commonlabtest

**Comment by reubenvarghese1** [comment]
@seekme94 , while you've put the module in the right category, there doesn't seem to be a downloadable omod . Can you please upload an omod?  For example, check this https://bintray.com/openmrs/omod/accesslogging out. Here , corresponding to a particular version of the module, there is a downloadable omod as well.

**Comment by reubenvarghese1** [comment]
@seekme94 , In case you're wondering how to do that, you just need to click on the version of your module after signing into your bintray account. Once there, you can see an option to upload files. It should look something like https://imagebin.ca/v/4XtxXqMwjvXt

**Comment by seekme94** [comment]
> @seekme94 , In case you're wondering how to do that, you just need to click on the version of your module after signing into your bintray account. Once there, you can see an option to upload files. It should look something like https://imagebin.ca/v/4XtxXqMwjvXt

But that's exactly what I did. Check here: https://bintray.com/seekme94/omod/commonlabtest/1.4.1#files

**Comment by reubenvarghese1** [comment]
Yes, that's what I was talking about. Thanks, this PR looks good to me now. You should be able to see your module in AddOns within the next 4 hrs :)

**Comment by seekme94** [comment]
> Yes, that's what I was talking about. Thanks, this PR looks good to me now. You should be able to see your module in AddOns within the next 4 hrs :)

Thanks @reubenvarghese1, I'll see if I can add more detail to the Module Release guide on Wiki.

**Pull request merged**